platform/src/main/java/org/stellar/anchor/platform/data/JdbcSep31TransactionStore.java

```diff
@@ -79,4 +85,32 @@ public JdbcSep31Transaction findByStellarAccountIdAndMemo(String accountId, Stri
   public Integer findByStatusCount(String status) {
     return transactionRepo.findByStatusCount(status);
   }
+
+  @Override
+  public List<? extends Sep31Transaction> findBulk(
+      TransactionsOrderBy orderBy,
+      Sort.Direction order,
+      @Nullable List<SepTransactionStatus> statuses,
+      int pageNumber,
+      int pageSize) {
+
+    Sort.Order primary =
+        order == Sort.Direction.ASC
+            ? Sort.Order.asc(orderBy.getTableName()).nullsLast()
+            : Sort.Order.desc(orderBy.getTableName()).nullsLast();
+    Sort sort = Sort.by(List.of(primary, Sort.Order.asc("id")));
+
+    PageRequest page = PageRequest.of(pageNumber, pageSize, sort);
```

Comment:
Why is this logic different from the SEP-24 method?

Reply:
I was just testing it out.
(Spoiler: doesn't work cause of https://github.com/spring-projects/spring-data-jpa/issues/1280 )